kentikapi/client_integration_test.go

```diff
@@ -5,32 +5,40 @@ import (
 	"encoding/json"
 	"fmt"
 	"io/ioutil"
+	"log"
+	"net"
 	"net/http"
 	"net/http/httptest"
 	"net/url"
 	"testing"
 	"time"
 
+	pb "github.com/kentik/api-schema-public/gen/go/kentik/synthetics/v202101beta1"
```

Comment:
I think that syntheticspb alias would be more informant. And there will be no need to rename it when we use couldexportpb package.

Reply:
Done